src/default_panels/GraphTransformsPanel.js

```diff
@@ -0,0 +1,84 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {
+  Radio,
+  TransformAccordion,
+  TraceAccordion,
+  DataSelector,
+  Dropdown,
+  Section,
+} from '../components';
+import {connectAggregationToTransform, localize} from '../lib';
+
+const AggregationSection = connectAggregationToTransform(Section);
+
+class UnlocalizedAggregations extends Component {
+  render() {
+    const {fullContainer: {aggregations = []}} = this.context;
+    const {localize: _} = this.props;
+    if (aggregations.length === 0) {
+      return null;
+    }
+
+    return aggregations.map(({target}, i) => (
+      <AggregationSection show key={i} aggregationIndex={i}>
+        <Dropdown
+          attr="func"
+          label={target}
+          options={[
+            {label: _('Count'), value: 'count'},
+            {label: _('Sum'), value: 'sum'},
+            {label: _('Average'), value: 'avg'},
+            {label: _('Median'), value: 'median'},
+            {label: _('Mode'), value: 'mode'},
+            {label: _('RMS'), value: 'rms'},
+            {label: _('Standard Deviation'), value: 'stddev'},
+            {label: _('Min'), value: 'min'},
+            {label: _('Max'), value: 'max'},
+            {label: _('First'), value: 'first'},
+            {label: _('Last'), value: 'last'},
```

Comment:
worth mapping over Plotly transform attributes so it is always in sync? Not a big deal I don't forsee them being updated often.

Reply:
That's not compatible with localization...